cpp/include/cugraph/algorithms.hpp

```diff
@@ -2033,6 +2035,13 @@ std::tuple<rmm::device_uvector<size_t>, rmm::device_uvector<vertex_t>> k_hop_nbr
   size_t k,
   bool do_expensive_check = false);
 
+// FIXME: Write doc string
+template <typename vertex_t, typename edge_t, typename weight_t, bool multi_gpu>
+rmm::device_uvector<vertex_t> compute_mis(
```

Comment:
And should we better rename this to `maximal_independent_set` to be consistent? We have `pagerank` instead of `compute_pagerank`.